pootle/static/js/editor/app.js

```diff
@@ -1575,8 +1575,11 @@ PTL.editor = {
     const suggestionIndex = suggestionTexts.indexOf(newTranslation);
 
     if (suggestionIndex !== -1 && !this.isFuzzy()) {
-      this.acceptSuggestion(suggestionIds[suggestionIndex], { skipToNext: true });
-      return;
+      if (this.settings.canReview) {
+        this.acceptSuggestion(suggestionIds[suggestionIndex], { skipToNext: true });
+        return;
+      }
+      this.selectedSuggestionId = suggestionIds[suggestionIndex];
```

Comment:
I would put here some comment saying something like "Exact match suggestions are still accepted this way for users with no review right".